docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -254,98 +250,281 @@ manhattan_job = define_asset_job(name="manhattan_job", selection="++manhattan_ma
+dagster asset list --select "taxi_zones_file+2"
+dagster asset materialize --select "taxi_zones_file+2"
 ```
 
 </TabItem>
 <TabItem value="dagster-ui" label="Dagster UI">
 
 ```shell
-taxi_zones_file*
+taxi_zones_file++
 ```
 
 Which would result in the following asset graph:
 
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/select-downstream-dependencies.png)
+![TODO]()
 
 </TabItem>
 </Tabs>
 
-#### Selecting a specific number of downstream layers \{#specific-downstream}
+### Select all upstream and downstream layers \{#full-lineage}
 
-To select an asset and multiple downstream layers, add plus sign (`+`) for each layer you want to select after the asset key in the query.
+To select an asset's entire lineage (all upstream and downstream layeres), add a plus sign (`+`) before and after the asset key in the query.
 
-This example selects the `taxi_trips_file` asset and two downstream layers.
+This example selects the entire lineage for the `taxi_zones` asset.
 
-<Tabs>
+<Tabs groupId="examples">
 <TabItem value="python" label="Python">
 
 ```python
-taxi_zones_job = define_asset_job(name="taxi_zones_job", selection="taxi_zones_file++")
+taxi_zones_all = AssetSelection.from_string("+taxi_zones+")
 ```
 
 </TabItem>
 <TabItem value="cli" label="CLI">
 
-When selecting an asset's dependencies using the CLI, enclose the plus sign (`+`) and the asset key in double quotes (`"`):
+When selecting an asset's entire lineage using the CLI, enclose the plus sign (`+`) and the asset key in double quotes (`"`):
 
 ```shell
-dagster asset list --select "taxi_zones_file++"
-dagster asset materialize --select "taxi_zones_file++"
+dagster asset list --select "+taxi_zones+"
+dagster asset materialize --select "+taxi_zones+"
 ```
 
 </TabItem>
 <TabItem value="dagster-ui" label="Dagster UI">
 
 ```shell
-taxi_zones_file++
++taxi_zones+
 ```
 
 Which would result in the following asset graph:
 
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/select-two-downstream-layers.png)
+![TODO]()
 
 </TabItem>
 </Tabs>
+
+## Filters, operands, and grouping
+
+Filters allow you to narrow your asset selection using specific criteria.
+
+| Filter | Syntax | Description |
+|--------|--------|-------------|
+| **Exact key** | `key:"my_key"` | Selects assets with the exact key `my_key`. |
+| **Key substring** | `key_substring:"partial_key"` | Selects assets whose key contains `partial_key`. |
+| **Tag (exact)** | `tag:"stage"` | Selects assets tagged with `stage`. |
+| **Tag (with value)** | `tag:"stage"="value"` | Selects assets tagged with `stage` having a specific `value`. |
+| **Owner** | `owner:"alice"` | Selects assets owned by `alice`. |
+| **Group** | `group:"team1"` | Selects assets in the group `team1`. |
+| **Kind** |  `kind:"table"` |  Selects assets of kind `table`. |
+| **Code location** | `code_location:"repo1"` | Selects assets located in code location `repo1`. |
+
```

Comment:
In Dagster Plus we additionally support the following filters:

`column_tag`
`columns`
`table_name`

Reply:
In the Gantt chart we also support a selection syntax:
<img width="542" alt="Screenshot 2025-02-20 at 1 52 57 PM" src="https://github.com/user-attachments/assets/41e29a17-bc96-4be9-9437-fd8a891e5eb5" />

This one only supports the attributes: `name`, `status`

Reply:
https://elementl.dagster.cloud/prod/runs/3ca30bf9-e2f3-4dad-8e8e-7c44159a792c

<img width="1728" alt="Screenshot 2025-02-20 at 1 54 53 PM" src="https://github.com/user-attachments/assets/70367a85-96d2-47e1-95a2-13f371fb0b69" />

Reply:
Full list in cloud:
<img width="814" alt="Screenshot 2025-02-20 at 10 02 45 AM" src="https://github.com/user-attachments/assets/a81a8cd1-ae69-4848-8b6e-74cb9428ef29" />


Additionally, in branch deployments we support the `changed_in_branch` filter, however we don't support `column`, `column_tag`, `table` in branch deployments.

We could add another column to the table "Supported views". And for each attribute we can list "OSS", "Dagster Plus", "Branch deployments".